sg_covid_impact/covid_data.py

```diff
@@ -0,0 +1,57 @@
+import re
```

Comment:
Docstring: """Fetch, save, and load Covid-19 case, death, and testing data from data.gov.uk."""